src/connector/src/sink/iceberg/mod.rs

```diff
@@ -261,13 +371,49 @@ impl Debug for IcebergSink {
 }
 
 impl IcebergSink {
+    async fn create_catalog(&self) -> Result<CatalogRef> {
```

Comment:
Should we use this method to unify the pulsar iceberg source and the iceberg sink?

Reply:
Yes, we will eventually need unify all these pieces together.